### PR TITLE
DTFS2-4871/4872 GEF-UI - Facility issued activity item

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made:

* Added activity for facility changing unissued -> issued in portal-api
* Added more fields to activity object
* Added maker object in facilities model (for when changed to issued)
* Changed rendering of activity timeline to be done in nunjucks due to HTML component
* Updated tests
* Updated e2e tests for activity objects
* Fixed mock-facilities for e2e tests where facility 1 had no ID
* Changed ukefFacilityId's to 1 2 3 4 for mock_facilities in gef e2e tests

### CODE REVIEW

File: gef-ui/server/controllers/activities-controller/index.test.js [hunk 1]
```diff
@@ -34,6 +64,38 @@ describe('mapPortalActivities', () => {
...
+    const response = mapPortalActivities(facilityActivity);
+
+    // expected format
+    const expected = [{
+      label: { text: 'Bank facility stage changed' },
+      text: '',
+      datetime: { timestamp: fromUnixTime(1638458265), type: 'datetime' },
+      byline: { text: 'Bob Smith' },
+      html: 'facility',
+      facilityType: 'Cash facility',
```
Comment (IgnatG) [inline gef-ui/server/controllers/activities-controller/index.test.js:87]: isn't this supposed to be `Cash`/`Contingent` etc?

Comment (Zainzzkk) [reply gef-ui/server/controllers/activities-controller/index.test.js:87]: that is for making a string, for the link so needs to have it as that

File: e2e-tests/gef/cypress/integration/application-details.spec.js [hunk 2]
```diff
@@ -148,7 +148,6 @@ context('Application Details Page', () => {
       applicationDetails.editRefNameLink().should('have.text', 'UKEF Test 123');
 
       statusBanner.bannerStatus().contains(CONSTANTS.DEAL_STATUS.DRAFT);
-      statusBanner.bannerExporter().should('not.contain', '-');
```
Comment (ttbarnes) [inline e2e-tests/gef/cypress/integration/application-details.spec.js:151]: Can we assert the exporter value?

Comment (Zainzzkk) [reply e2e-tests/gef/cypress/integration/application-details.spec.js:151]: done

File: gef-ui/server/controllers/mocks/index.js [hunk 3]
```diff
@@ -277,6 +277,9 @@ const MockRequestUnissued = () => {
     user: {
       bank: { id: 'BANKID' },
       roles: ['MAKER'],
+      _id: 12345,
```
Comment (ttbarnes) [inline gef-ui/server/controllers/mocks/index.js:280]: v.minor but should be a string to be consistent

Comment (Zainzzkk) [reply gef-ui/server/controllers/mocks/index.js]: done

File: portal-api/src/constants/deal.js [hunk 4]
```diff
@@ -62,4 +68,5 @@ module.exports = {
   SUBMISSION_TYPE,
   ACTION_NAME,
   DEAL_STATUS,
+  SUBMISSION_TYPE_LOWERCASE,
```
Comment (IgnatG) [inline portal-api/src/constants/deal.js:71]: where is this being used?

Comment (Zainzzkk) [reply portal-api/src/constants/deal.js:71]: removed, I was going to use this elsewhere but forgot to remove it

**Review by IgnatG — APPROVED** [review]

**Pull request merged**